Modules/Core/Repositories/Eloquent/EloquentBaseRepository.php

```diff
@@ -191,4 +191,26 @@ public function clearCache()
     {
         return true;
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function where($field, $value, $operator=null)
+    {
+        if (null==$operator) {
+            $operator='=';
```

Comment:
Same here, spaces.
``` php
        if (null == $operator) {
            $operator = '=';
```

Reply:
Should also be strict checking `null`: `if ($operator === null) {`

(As a side note, why do people still use Yoda syntax? It makes things harder to parse...)

Reply:
I think that many people use it because are safer, image to forgot a `=`, with `null = $variable` you have an error at runtime

Reply:
That's why I always use strict (`===`) over loose (`==`) comparison... Pretty easy to spot two missing equals signs instead of just one.

Reply:
I just use the `!` but I prefer strict too over the loose comparison

Reply:
Indeed, strict comparison is preferred.